frame/vesting/src/lib.rs

```diff
@@ -181,6 +217,13 @@ pub mod module {
 		ValueQuery,
 	>;
 
+	/// Counter used to uniquely identify vesting schedules within this pallet.
+	#[pallet::storage]
+	#[pallet::getter(fn vesting_schedules_count)]
+	#[allow(clippy::disallowed_types)] // nonce, ValueQuery is OK
+	pub type VestingScheduleCount<T: Config> =
```

Comment:
I would rename this to `VestingScheduleNonce` (or something similar)

Reply:
✅